docs/Model/Result.md

```diff
@@ -13,7 +13,7 @@ Name | Type | Description | Notes
 **status** | **string** |  | [optional]
 **isApiResult** | **bool** |  | [optional]
 **timeSpentMs** | **int** |  | [optional]
-**endTime** | **\DateTime** |  | [optional]
+**endTime** | **string** | date in the &#39;Y-m-d H:i:s&#39; format | [optional]
```

Comment:
we discussed about it at March 25th

our decision was to make a research first, and then fix the api.

Reply:
fixed